src/components/AccessControlNew.tsx

```diff
@@ -73,12 +80,14 @@ const AccessControlNew = () => {
         return {
             ID: formRule.ID,
             Name: formRule.Name,
+            Description: formRule.Description,
```

Comment:
Codacy has a fix for the issue: [Delete `······`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      Description: formRule.Description,
```